src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_staticapp_commands_thru_mock.py

```diff
@@ -296,17 +296,17 @@ def test_list_staticsite_function_app_settings_without_resourcegroup(self):
             self.rg1, self.name1)
 
     def test_set_staticsite_function_app_settings_with_resourcegroup(self):
-        app_settings1_input = ['key1=val1', 'key2=val2']
-        app_settings1_dict = {'key1': 'val1', 'key2': 'val2'}
+        app_settings1_input = ['key1=val1', 'key2=val2==', 'key3=val3=']
```

Comment:
Can a key contain `=`?

Reply:
Good question. According to the REST output the answer is no. Here's what I got:


```
Bad Request({"Code":"BadRequest","Message":"appSettings is invalid.  App setting key can only consist of alphanumeric characters, '.', and '_'. foo= is invalid.","Target":null,"Details":[{"Message":"appSettings is invalid.  App setting key can only consist of alphanumeric characters, '.', and '_'. foo= is invalid."},{"Code":"BadRequest"},{"ErrorEntity":{"ExtendedCode":"51021","MessageTemplate":"{0} is invalid.  {1}","Parameters":["appSettings","App setting key can only consist of alphanumeric characters, '.', and '_'. foo= is invalid."],"Code":"BadRequest","Message":"appSettings is invalid.  App setting key can only consist of alphanumeric characters, '.', and '_'. foo= is invalid."}}],"Innererror":null})
```